src/App.js

```diff
@@ -20,55 +20,66 @@ function Board({ xIsNext, squares, onPlay }) {
     } else {
       nextSquares[i] = "O";
     }
-    onPlay(nextSquares);
+    onPlay(nextSquares, i);
   }
-  const winner = calculateWinner(squares);
+
+  const winnerData = calculateWinner(squares);
   let status;
```

Comment:
I would change variable's name
```suggestion
  let gameStatus;
```

Reply:
Done.